projects/commudle-admin/src/app/feature-modules/events/components/create-event/create-event.component.ts

```diff
@@ -103,8 +110,9 @@ export class CreateEventComponent implements OnInit {
 
   setStartDateTime() {
     this.startDate = this.eventForm.get('event').get('start_date').value;
-    this.startHour = this.eventForm.get('event').get('start_hour').value;
-    this.startMinute = this.eventForm.get('event').get('start_minute').value;
+    this.startHour = this.eventForm.get('event').get('start_time_pick').value.getHours();
+    this.startMinute = this.eventForm.get('event').get('start_time_pick').value.getMinutes();
```

Comment:
Probably we can store this in a single variable and get hours and minutes from it?